content/docs/tasks/traffic-management/egress/index.md

```diff
@@ -422,6 +439,14 @@ However, configuring the proxy this way does require cluster provider specific k
 In addition to that, you loose monitoring of access to external services and cannot apply Istio features on traffic to
 external services.
 
+The third approach directs the traffic through the Istio sidecar proxy, however it allows access to any service on any
+ports that have no HTTP service inside the mesh and no service entry defined. In this approach, similarly to the
+previous one, you cannot monitor access to external services. The advantage of this approach is that you must not
+know which IP ranges are external to the cluster. Additional advantage is that you can decide to switch to the first
+approach per specific port, by creating service entries for that port. This way you can start Istio by allowing access
+to any external service and then decide to start controlling access to external services per specific port, gaining
```

Comment:
for specific ports, enabling monitoring and ...

Reply:
Fixed in https://github.com/istio/istio.io/pull/3222/commits/df7ecc5dc81d43193267ed33c7d43f32c099d950